modules/browser/BrowserCapabilities.js

```diff
@@ -16,49 +18,11 @@ export default class BrowserCapabilities extends BrowserDetection {
      *
      * @param {boolean} [isUsingIFrame] - True if Jitsi Meet is loaded in iframe
```

Comment:
Update comments? Were these constructor arguments moved into js-utils BrowserDetection?

Reply:
Yes! Will fix it!